crates/bevy_math/src/interpolation/utils.rs

```diff
@@ -0,0 +1,67 @@
+use std::ops::{Add, Div, Mul, Sub};
+
+/// Quake 3 fast inverse sqrt
+///
+/// Took this one from piston: https://github.com/PistonDevelopers/skeletal_animation
+#[inline]
+pub fn inv_sqrt(x: f32) -> f32 {
```

Comment:
```suggestion
pub fn fast_inv_sqrt(x: f32) -> f32 {
```
This should be called out as approximate IMO.

Reply:
done